### ISSUE TITLE
Accessibility: Screen readers (Narrator, JAWS, NVDA) read aloud "DesktopChildSiteBridge" on XAML Windows

### ISSUE THREAD

**Issue by rmlaapl** [microsoft/microsoft-ui-xaml#9352]
**TL;DR**
Applications that speak out loud what is happening on the screen (for sight-impaired people) appear to use the window's text as the source of words that they speak whenever Automation/Accessibility properties are not set.

The following is a problem in at least version `1.4.2312190001` of Windows App SDK: an SDK-created window with the class name _"Microsoft.UI.Content.ContentWindowSiteBridge"_ has its text set (i.e. WM_SETTEXT) to `DesktopChildSiteBridge`.  This causes a screen reader to read the title of your top-level XAML window, and then immediately speak that string of words, which would be very disorienting as a person relying on accessibility features.

This is a screenshot from Accessibility Insights, highlighting the window in question:

<img width="369" alt="Screenshot 2024-02-21 at 19 19 01" src="https://github.com/microsoft/microsoft-ui-xaml/assets/160653809/3da91969-2eda-49da-8db4-d89f4d171543">

The fix for this is extremely simple: just don't set the window text on that ContentWindowSiteBridge, and all will be well :)

I noticed this was/is also an [issue in MAUI](https://github.com/dotnet/maui/issues/3284)

In our application, I fixed it with some old school Win32: `EnumChildWindows` looking for that class name, and if found, set its text to an empty string.

**Comment by bpulliam** [comment]
not a duplicate of #7853. that was fixed in 1.3. [internal bug](http://task.ms/41935066)
